tests/testthat/test_featsel_filters.R

```diff
@@ -73,6 +71,13 @@ test_that("randomForestSRC_var.select filter handles user choices correctly", {
   )
 })
 
+test_that("randomForestSRC_var.select minimal depth filter returns NA for features below the threshold", {
+  dat = generateFilterValuesData(task = multiclass.task, 
+                                 method = "randomForestSRC_var.select",
+                                 nselect = 5,
+                                 more.args = list("randomForestSRC_var.select" = list(method = "md", nrep = 5)))
```

Comment:
Thanks.

Please style the changed files according to our [mlr-style](https://github.com/mlr-org/mlr3/wiki/style-guide#styler-mlr-style).

Reply:
OK done.